src/System.Runtime.Extensions/tests/System/Environment.SetEnvironmentVariable.cs

```diff
@@ -85,6 +86,7 @@ public void Default(EnvironmentVariableTarget target)
 
 
         [Theory]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap | TargetFrameworkMonikers.UapAot, "SetEnvironmentVariableW isn't supported in Appx.")] // Issue #19870
```

Comment:
SetEnvironmentVariable() is now brought up on .NetNative and I'm bringing up the Target version today. Disabling these now will impede my work.

Can we change these disables to a runtime check ("IsSetEnvironmentVariableAvailable")  that checks if you're running in an appcontainer instead?